airflow/jobs/scheduler_job.py

```diff
@@ -1265,6 +1265,7 @@ def _process_executor_events(self, simple_dag_bag, session=None):
                     msg = ("Executor reports task instance {} finished ({}) "
                            "although the task says its {}. Was the task "
                            "killed externally?".format(ti, state, ti.state))
+                    Stats.incr('scheduler.tasks.killed_externally', 1, 1)
```

Comment:
```suggestion
                    Stats.incr('scheduler.tasks.killed_externally')
```

Those two parameters are default.